mkosi/resources/mkosi-initrd/mkosi.conf.d/10-serpentos.conf

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+# Completely override the default Packages
+Packages=
+Packages=
+        systemd
+        dash
+        # Various libraries that are dlopen'ed by systemd
+
+        # File system checkers for supported root file systems
+        e2fsprogs
+        xfsprogs
+
+        # fsck.btrfs is a dummy, checking is done in the kernel.
+
+        util-linux
+
+RemoveFiles=
+        /.moss
+
+        # Remove all files that are only required for development.
+        /usr/bin/mkfs*
+        /usr/bin/mke2fs
+        /usr/lib/*.a
+        /usr/lib/kernel/*
+
+        /usr/share/bash-completion/*
```

Comment:
How much does this actually save? We don't remove bash completions on any other distributions either

Reply:
In that case please keep these in there for now.

> The resulting initrd is still 2.5x the original SOS one.

That's probably because we include all kernel modules by default which SOS might not do. The rest if probably simply due to moss not doing sufficient subpackages, causing lots of unnecessary stuff to be pulled into the initrd. This is something to solve within SOS itself.